From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: MiniDigger | Martin <admin@benndorf.dev>
Date: Sun, 20 Oct 2024 09:24:36 +0200
Subject: [PATCH] Feature


diff --git a/Test.java b/Test.java
index bf731c042bf7f59b60378df1d61302e1769a6a77..13e57904b05e4b98b8b105511254c2f6c1b1924d 100644
--- a/Test.java
+++ b/Test.java
@@ -13,4 +13,8 @@ public class Test {
     public final String getTest2() {
         return this.test + "2";
     }
+
+    public String feature() {
+        return "amazing feature";
+    }
 }
