--- a/net/minecraft/CrashReport.java
+++ b/net/minecraft/CrashReport.java
@@ -35,10 +_,11 @@
     public CrashReport(String title, Throwable exception) {
         this.title = title;
         this.exception = exception;
+        // FIXME this.systemReport.setDetail("CraftBukkit Information", new org.bukkit.craftbukkit.CraftCrashReport()); // CraftBukkit
     }
 
     private final String getTitle() {
-        return this.title;
+        return this.title; // Test
     }
 
     public Throwable getException() {
